bazel_tools/scala.bzl

```diff
@@ -414,6 +415,29 @@ def da_scala_library(name, **kwargs):
                 )
                 break
 
+def da_scala_library_suite(name, **kwargs):
+    """
+    Define a suite of Scala libraries as a single target.
+
+    Applies common Scala options defined in `bazel_tools/scala.bzl`.
+    And forwards to `scala_library_suite` from `rules_scala`.
+    Refer to the [`rules_scala` documentation][rules_scala_library_suite_docs].
+
+    [rules_scala_library_suite_docs]: https://github.com/bazelbuild/rules_scala/blob/master/docs/scala_library_suite.md
+    """
+    _wrap_rule(scala_library_suite, name, **kwargs)
+    _create_scala_source_jar(name = name, **kwargs)
+    _create_scaladoc_jar(name = name, **kwargs)
+
+    if "tags" in kwargs:
+        for tag in kwargs["tags"]:
+            if tag.startswith("maven_coordinates="):
+                pom_file(
+                    name = name + "_pom",
+                    target = ":" + name,
+                )
+                break
```

Comment:
Unfortunately, this won't work. [`scala_library_suite`](https://github.com/bazelbuild/rules_scala/blob/f1e689c2426c1522d7c8ca69ac8cbc1a714b6d02/scala/private/rules/scala_library.bzl#L168-L191) doesn't forward `tags` to the one final collection `scala_library`, but instead to the many individual `scala_library` in the suite.

We'd either need to patch `scala_library_suite` to allow setting tags on the collection library, or replicate the [name sanitizing logic](https://github.com/bazelbuild/rules_scala/blob/f1e689c2426c1522d7c8ca69ac8cbc1a714b6d02/scala/private/rules/scala_library.bzl#L176) here and generate a `pom_file` for every individual library. 
However, I think it's fair to not support `pom_file` on `da_scala_library_suite` for now. As there's no use-case for it, yet. However, it would be good to throw an error in case somebody tries to use it.

Reply:
Like this? https://github.com/digital-asset/daml/pull/3486/commits/b39b1f48544f24bbbc6bbb09e36ddb0ab938c69d

Reply:
Yes, thank you!

Reply:
No, thank you!